src/transformers/models/albert/configuration_albert.py

```diff
@@ -103,51 +109,51 @@ class AlbertConfig(PretrainedConfig):
     >>> configuration = model.config
     ```"""
 
+    vocab_size: int = validated_field(interval(min=1), default=30000)
```

Comment:
is there a way to juste write `validate` or `check` instead of `check_field`?

Reply:
(cc @Wauplin)